openfga/v1/openfga_service.proto

```diff
@@ -672,34 +677,40 @@ service OpenFGAService {
     };
   }
 
-  // StreamedListObjects is a streaming variation of the ListObjects API (see ListObjects for more info).
   rpc StreamedListObjects(StreamedListObjectsRequest) returns (stream StreamedListObjectsResponse) {
     option (google.api.http) = {
       post: "/stores/{store_id}/streamed-list-objects",
       body: "*"
     };
 
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "[EXPERIMENTAL] Streaming variation of the ListObjects API (see ListObjects for more info)"
       tags: [
         "Relationship Queries"
       ]
       operation_id: "StreamedListObjects"
     };
   }
 
-  // ListObjects lists all of the object ids for objects of the provided type that the given user
-  // has a specific relation with.
   rpc ListObjects(ListObjectsRequest) returns (ListObjectsResponse) {
     option (google.api.http) = {
       post: "/stores/{store_id}/list-objects",
       body: "*"
     };
 
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "[EXPERIMENTAL] Get all object ids of the given type that the given user has a given relation with"
       tags: [
         "Relationship Queries"
       ]
       operation_id: "ListObjects"
+      description:
+        "The ListObjects API returns a list of all the object ids of the given type that the given user "
+        "has a given relation with. To achieve this, both the store tuples and the authorization model are used.\n"
```

Comment:
Don't need the line "To achieve this, both the store tuples and the authorization model are used"

Reply:
I put it there so that the line `You may also specify `contextual_tuples` that will be treated as regular tuples.` makes sense 🤔